packages/components/src/panel/README.md

```diff
@@ -127,6 +127,20 @@ An icon to be shown next to the `PanelBody` title.
 -   Type: `String`
 -   Required: No
 
+###### iconOpen
+
+An icon to be shown as the toggle button within the `PanelBody` title when open.
+
+-   Type: `String`
+-   Required: No
+
+###### iconClosed
```

Comment:
Let's use "collapsed" instead of "closed"

Reply:
Changed to `iconCollapsed`